package.json

```diff
@@ -26,17 +26,16 @@
     "@angular/common": ">=4.3.6",
     "@angular/core": ">=4.3.6",
     "@angular/forms": ">=4.3.6",
-    "@skyux/forms": "^3.0.0-rc.0",
-    "@skyux/i18n": "^3.3.0",
-    "@skyux/list-builder": "^3.0.0-rc.0",
-    "@skyux/list-builder-common": "^3.0.0-rc.1",
+    "@skyux/i18n": "^3.5.0",
+    "@skyux/forms": "^3.0.0",
```

Comment:
Looks like `forms` and `i18n` aren't in alphabetical order?

Reply:
reordered